### PR TITLE
Ref [TASK][39888] Checkout page - auth completed.

### PR DESCRIPTION
**Pull request by hi-Long**
- Link redmine: https://edu-redmine.sun-asterisk.vn/issues/39888
- Completed UI:
Not authenticated:
screencapture-localhost-3000-checkout-auth-2021-08-24-08_41_11
Authenticated:
screencapture-localhost-3000-checkout-auth-2021-08-24-08_41_43

### CODE REVIEW

**Comment by hi-Long** [comment]
ready

File: src/helpers/convertDate.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    let yyyy = today.getFullYear();
+    if (dd < 10) {
+        dd = '0' + dd;
+    }
+
+    if (mm < 10) {
+        mm = '0' + mm;
+    }
+    today = dd + '/' + mm + '/' + yyyy;
+    return today
```
Comment (manhnt-0960) [inline src/helpers/convertDate.js:16]: tham khảo thư viện này e nhé
https://formatjs.io/docs/react-intl/components/#formatteddate

File: src/helpers/formatPrice.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const formatPrice = (x) => {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".");
```
Comment (manhnt-0960) [inline src/helpers/formatPrice.js:2]: tham khảo e nhé
https://formatjs.io/docs/react-intl/components/#formattednumber

**Comment by hi-Long** [comment]
ready

**Pull request merged**